modules/ROOT/pages/message-logging-policy.adoc

```diff
@@ -85,6 +85,37 @@ Note: If the payload is used as part of the message log, do not configure the Li
 |After Calling API |Set to cause your logging policy to log messages after the API endpoint is called, taking into account the order in which this policy is applied |Optional
 |===
 
+== Example of Logging Policy from the UI
+
+To better understand how to use the Message Logging policy, consider an example "Les Vetments" company that is in the business of retail clothing and more. To run their e-commerce portal successfully, Les Vetments needs to audit every event and transaction made on their website.
+
+For example, if a customers buys or returns a product, or if the company adds or removes a product from the catalogue, the transactions must be logged. The IT manager at Les Vetments applies the Message Logging policy to their APIs to enable logging and viewing any data or transaction changes without making any modifications in the code.
+
+Les Vetments uses the resource-level policy's granularity to apply the policy to only `POST`, `PUT`, `PATCH` and `DELETE` methods. To log all events, Les Vetments applies the Message Logging policy to the API with the following UI configurations:
+
+*Message*: #[payload]
```

Comment:
Correction here, the expression to use is #[message], and everything in the message will be logged. We can then add the following examples to show other alternatives:

#[attributes.headers] if the requirement is to log the headers
OR
#['User ' ++ authentication.clientId ++ ' performed action ' ++ attributes.method ++ ' on ' ++ attributes.requestPath ++ ' with Payload: ' ++ payload]
It's an example where there is more customization in the information to log, in this case you see the user, action being performed and the body.

Reply:
Done! Thanks!